server/vcr-server/vcr_server/middleware/routing.py

```diff
@@ -39,7 +39,10 @@ def __init__(self, get_response):
         self.get_response = get_response
 
     def __call__(self, request):
-        self.process_request(request)
+        (request, _requested_version, _request_path_info) = self.process_request(request)
```

Comment:
Not sure why this was changed

Reply:
this was changed by the merge pull request. I checked and both changes are already in main:
- https://github.com/bcgov/aries-vcr/blob/403da51a1b758918c8f30f48290d989f65982910/server/vcr-server/vcr_server/middleware/routing.py#L41
- https://github.com/bcgov/aries-vcr/blob/403da51a1b758918c8f30f48290d989f65982910/docker/docker-compose.yml#L70

Reply:
merge pull fixed this issue